hack/fuse-demo/coord-app-pg.Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM debian
+
+RUN apt-get update &&\
+    curl -sL https://deb.nodesource.com/setup_10.x  | bash &&\
+    apt-get install -y \
+        curl \
+        postgresql \
```

Comment:
We need to lock down the version of postgres and the build itself. We can store the deb file within datamon.

Reply:
https://www.postgresql.org/docs/current/upgrading.html